### PR TITLE
[Kotlin migration] ReviewerTest

### PR DESCRIPTION
**Pull request by BrayanDSO**
## Checklist
_Please, go through these checks before submitting the PR._

- [X] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [X] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/anki/ReviewerTest.kt [hunk 1]
```diff
@@ -13,458 +13,424 @@
...
+        val col = col
+        val nw = col.decks.confForDid(1).getJSONObject("new")
+        val time = col.time as MockTime
+        nw.put("delays", JSONArray(intArrayOf(1, 10, 60, 120)))
 
-        Card[] cards = new Card[4];
-        cards[0] = addRevNoteUsingBasicModelDueToday("1", "bar").firstCard();
-        cards[1] = addNoteUsingBasicModel("2", "bar").firstCard();
-        cards[2] = addNoteUsingBasicModel("3", "bar").firstCard();
+        val cards = arrayOfNulls<Card>(4)
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/anki/ReviewerTest.kt:195]: kotlin cleanup: can this be non-null?

**Review by david-allison — CHANGES_REQUESTED** [review]
Awesome

File: AnkiDroid/src/test/java/com/ichi2/anki/ReviewerTest.kt (hunk 1 above)
Comment (BrayanDSO) [reply AnkiDroid/src/test/java/com/ichi2/anki/ReviewerTest.kt:196]: Yep. Nice catch!

**Review by mikehardy — APPROVED** [review]
That's a big one! LGTM

**Review by david-allison — APPROVED** [review]

**Pull request merged**